crates/driver/src/domain/competition/auction.rs

```diff
@@ -211,10 +217,17 @@ impl AuctionProcessor {
                 order::Kind::Limit { .. } => 1,
                 order::Kind::Liquidity => 0,
             };
+            let creation_timestamp = order.quote_solver.filter(|s| &s.0 == solver).and_then(|_| {
+                order
+                    .created_on
+                    .filter(|created_on| *created_on > (Utc::now() - Duration::minutes(2)))
```

Comment:
Not sure if it's worth creating a new config arg for that.

Reply:
I think we should create different sorting strategies that we can configure for each driver individually. The ones I can think off the top of my head:
1. external price (status quote)
2. creation timestamp (regardless of quote)
3. own quotes (within some hardcoded time, no need to customize I believe), followed by external price

Let's abstract that part so that we can experiment with different sorting strategies.

Reply:
> external price (status quote)

Is that about the likelihood or something else?

Reply:
An attempt to implement it: https://github.com/cowprotocol/services/pull/2855/commits/492ef0dd0761f2b4b4f028806349dda5736bb445